test/system/200-pod.bats

```diff
@@ -205,6 +205,7 @@ function random_ip() {
     # entrypoint to confirm that --infra-command will override.
     local infra_image="infra_$(random_string 10 | tr A-Z a-z)"
     local infra_command="/pause_$(random_string 10)"
+    local infra_name="testing-infra-container"
```

Comment:
I realize I'm late in the game and that tests are green. If you have to resubmit for some reason, please fix this so `infra_name` includes a random element. Randomization is critical in testing, it helps catch all manner of subtle bugs.